orientation.js

```diff
@@ -299,7 +297,6 @@ var applyOrientationToPage = function(page, args){
 
 
 	page._refreshCss();
```

Comment:
Is this still needed?

`nativescript-dom` should already update the styling when page.classList is update (or at least what's how I understand the code)